multiqc/modules/multivcfanalyzer/multivcfanalyzer.py

```diff
@@ -0,0 +1,244 @@
+            'format': '{:,.0f}'
+        }
+        headers['discardedVarCall'] = {
+            'title': 'Discarded SNP Call',
+            'description': 'Number of non-reference positions not reaching genotyping quality threshold',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['filteredVarCall'] = {
+            'title': 'Filtered SNP Call',
+            'description': 'Number of positions ignored defined in user-supplied filter list',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['refCall'] = {
+            'title': 'Number of Reference Calls',
+            'description': 'Number of reference calls made',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['discardedRefCall'] = {
+            'title': 'Discarded Reference Call',
+            'description': 'Number of reference positions not reaching genotyping quality threshold',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['noCall'] = {
+            'title': 'Positions with No Call',
+            'description': 'Number of positions with no call made as reported by GATK',
+            'scale': 'BuPu',
+            'shared_key': 'calls'
+        }
+        headers['coverage (fold)'] = {
+            'title': 'SNP Coverage',
+            'description': 'Average number of reads covering final calls',
+            'scale': 'OrRd',
+            'shared_key': 'coverage',
+            'suffix': 'X'
+        }
+        headers['coverage (percent)'] = {
+            'title': '% SNPs Covered',
+            'description': 'Percent coverage of all positions with final calls',
+            'scale': 'PuBuGn',
+            'shared_key': 'coverage',
+            'suffix': '%'
+        }
+        headers['unhandledGenotype'] = {
+            'title': 'Unhandled Genotypes',
+            'description': 'Number of positions discarded due to presence of more than one alternate allele',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'snp_count'
+        }
+
+        #Separate table config
+        table_config = {
+        'namespace': 'MultiVCFAnalyzer',                         # Name for grouping. Prepends desc and is in Config Columns modal
+        'id': 'mvcf-table',                 # ID used for the table
+        'table_title': 'MultiVCFAnalyzer Results',             # Title of the table. Used in the column config modal
+        }
+        tab = table.plot(self.mvcf_data, headers, table_config)
+        return tab
+
+
+    
+    def addBarplot(self):
+        """ Take the parsed stats from MultiVCFAnalyzer and add it to the MVCF Table"""
+        cats = OrderedDict()
+
+        cats['SNP Calls (all)'] = {
+            'name': 'SNP Calls (all)',
```

Comment:
I guess the `(all)` shouldn't be included here?  I need to clarify this with Alexander I think